public/app/features/dashboard/components/PanelEditor/PanelEditorTabs.tsx

```diff
@@ -40,6 +40,7 @@ export const PanelEditorTabs: React.FC<PanelEditorTabsProps> = ({ panel, dashboa
               key={tab.id}
               label={tab.text}
               active={tab.active}
+              counter={5}
```

Comment:
let's not forget to remove this ;)

Reply:
ah yes, would be good to change that the real value

Reply:
Very good catch!